test/fixtures/tab.ftl

```diff
@@ -0,0 +1,14 @@
+# Error (tab after =)
+key01 =	Value 01
```

Comment:
I think this should be a success.

Reply:
Should the tab be trimmed from the value or kept verbatim?

Reply:
I guess it makes sense to only trim syntax whitespace. I'd keep it.